src/app/modules/home/components/welcome/welcome.component.html

```diff
@@ -0,0 +1,6 @@
+<nav class="navbar" *ngIf="currentUser">
+    <div class="navbar-nav">
+        <a class="nav-item nav-link">Home</a>
```

Comment:
if we gonna using BEM, so:
- navbar-nav should be navbar__inner 
- 'nav-item' should be 'nav__item'
- nav-link should be 'nav__link'
If you use 'nav-item' its mean, that follow block gonna have a difficult structure, it's independence and can be reusable

Reply:
And also, better use ul>li inside nav, to make in write semantically